iron-validator-behavior.html

```diff
@@ -52,11 +52,8 @@
      *
      * @method validate
```

Comment:
The compiler doesn't use @method annotations at all, so unless it's useful for the doc generator, feel free to get rid of them.